### ISSUE TITLE
Typescript Typings - SdkError Not Surfaced

### ISSUE THREAD

**Issue by bdbvb** [MyPureCloud/genesys-cloud-webrtc-sdk#618]
I'm using Typescript. To reference the SdkError type, I'm having to drill down into the distribution [and the path to that changed from 6.0.1 to 6.1.0].  See below. Any way to surface that and other types at the top level?

Thanks!

```
import { GenesysCloudWebrtcSdk, SessionTypes } from "genesys-cloud-webrtc-sdk";
import { SdkError } from "genesys-cloud-webrtc-sdk/dist/es/utils";
...
const client = new GenesysCloudWebrtcSdk({...});
...
client.on("sdkError", (error: SdkError) => {...});
...
```

**Comment by jensengar** [comment]
@bdbvb we've noticed this as well. We'll work on getting a fix out, but I can't promise a timeframe. Thanks for letting us know!

**Comment by djhouseknecht** [comment]
PR #621 will solve this. Once it merges, we will get it released in v6.1.1 to fix it.

**Comment by jensengar** [comment]
v6.1.1 has been released.
